packages/core/contracts/oracle/implementation/VotingToken.sol

```diff
@@ -22,23 +22,15 @@ contract VotingToken is ExpandedERC20, ERC20Snapshot {
         return _snapshot();
     }
 
-    // _transfer, _mint and _burn are ERC20 internal methods that are overridden by ERC20Snapshot,
-    // therefore the compiler will complain that VotingToken must override these methods
-    // because the two base classes (ERC20 and ERC20Snapshot) both define the same functions
+    // _beforeTokenTransfer is an ERC20 internal method that is overridden by ERC20Snapshot,
+    // therefore the compiler will complain that VotingToken must override this method
+    // because the two base classes (ERC20 and ERC20Snapshot) both define the same function
 
-    function _transfer(
+    function _beforeTokenTransfer(
```

Comment:
Ideally we shouldn't change VotingToken overrides, since this wouldn't be reflective of the UMA token implementation on chain.

Why is this change necessary?